### ISSUE TITLE
The matchTextDirection not work. 

### ISSUE THREAD

**Issue by omidraha** [dnfield/flutter_svg#414]
The `     matchTextDirection: true,` attribute not works for `RTL` locales.

This  works:

```dart
import 'package:flutter/material.dart';
import 'package:flutter/cupertino.dart';
import 'package:flutter_svg/svg.dart';
import 'package:flutter_localizations/flutter_localizations.dart';

void main() {
  runApp(Sample());
}

class Sample extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      debugShowCheckedModeBanner: false,
      localizationsDelegates: [
        GlobalCupertinoLocalizations.delegate,
        GlobalMaterialLocalizations.delegate,
        GlobalWidgetsLocalizations.delegate,
      ],
      supportedLocales: [
        Locale('en', 'US'),
        Locale('fa', 'IR'),
      ],
      locale: Locale('en', 'US'),
      home: SafeArea(
        child: Scaffold(
          body: Row(
            mainAxisAlignment: MainAxisAlignment.start,
            children: [
              Text('Text one'),
              SvgPicture.asset(
                'assets/images/arrow_right.svg',
                matchTextDirection: true,
              ),
              Text('Text two'),
            ],
          ),
        ),
      ),
    );
  }
}
```
Output:

![matchTextDirection_ltr](https://user-images.githubusercontent.com/397672/92381360-f2eb2b80-f11f-11ea-8561-ee0f1f188048.png)

But when I changed `locale` to:

```dart
locale: Locale("fa", "IR"),
```
as this code:
```dart
import 'package:flutter/material.dart';
import 'package:flutter/cupertino.dart';
import 'package:flutter_svg/svg.dart';
import 'package:flutter_localizations/flutter_localizations.dart';

void main() {
  runApp(Sample());
}

class Sample extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      debugShowCheckedModeBanner: false,
      localizationsDelegates: [
        GlobalCupertinoLocalizations.delegate,
        GlobalMaterialLocalizations.delegate,
        GlobalWidgetsLocalizations.delegate,
      ],
      supportedLocales: [
        Locale('en', 'US'),
        Locale('fa', 'IR'),
      ],
      locale: Locale("fa", "IR"),
      home: SafeArea(
        child: Scaffold(
          body: Row(
            mainAxisAlignment: MainAxisAlignment.start,
            children: [
              Text('Text one'),
              SvgPicture.asset(
                'assets/images/arrow_right.svg',
                matchTextDirection: true,
              ),
              Text('Text two'),
            ],
          ),
        ),
      ),
    );
  }
}

```
The image is not shown.

Output:

![matchTextDirection_rtl](https://user-images.githubusercontent.com/397672/92381627-786edb80-f120-11ea-8dc8-7f6778099c71.png)

Info:
The `arrow_right.svg` file:
```svg
<svg xmlns="http://www.w3.org/2000/svg" width="18.914" height="13.828" viewBox="0 0 18.914 13.828">
  <g id="Group_35117" data-name="Group 35117" transform="translate(2999 1993.914) rotate(180)">
    <line id="Line_70" data-name="Line 70" x1="5.5" y2="5.5" transform="translate(2981.5 1981.5)" fill="none" stroke="#d0d0d0" stroke-linecap="square" stroke-width="2"/>
    <line id="Line_71" data-name="Line 71" x2="5.5" y2="5.5" transform="translate(2981.5 1987)" fill="none" stroke="#d0d0d0" stroke-linecap="square" stroke-width="2"/>
    <line id="Line_72" data-name="Line 72" x2="15.125" transform="translate(2982.875 1987)" fill="none" stroke="#d0d0d0" stroke-linecap="square" stroke-width="2"/>
  </g>
</svg>

```

**Comment by dnfield** [comment]
Transform is probably not getting handled properly that we try to apply for RTL.

**Comment by omidraha** [comment]
I request a PR #416  to solve this problem, 
I also tested with [svgStr](https://github.com/dnfield/flutter_svg/blob/2408b057c59fac7c93523c2b4b15734550bae220/test/widget_svg_test.dart#L21) and [stickFigureSvgStr](https://github.com/dnfield/flutter_svg/blob/2408b057c59fac7c93523c2b4b15734550bae220/test/widget_svg_test.dart#L43) values.

Here are output:

Before PR: 
  1. `matchTextDirection: false` :  [Result1](https://user-images.githubusercontent.com/397672/92923720-1f1ee900-f44d-11ea-89ef-76692cde99c2.png), [Result2](https://user-images.githubusercontent.com/397672/92924077-a53b2f80-f44d-11ea-945a-f9aade4160e8.png) 
  2. `matchTextDirection: true`:   [Result1](https://user-images.githubusercontent.com/397672/92923754-2b0aab00-f44d-11ea-8f3e-72b8da1016a2.png), [Result2](https://user-images.githubusercontent.com/397672/92924118-ad936a80-f44d-11ea-8969-37934e61aa0c.png) 

After PR: 
  1. `matchTextDirection: false`:  [Result1](https://user-images.githubusercontent.com/397672/92924513-54780680-f44e-11ea-9c6f-63d8fddbe6fc.png), [Result2](https://user-images.githubusercontent.com/397672/92924770-b6387080-f44e-11ea-9b33-29f25f543508.png) 
  2. `matchTextDirection: true` :  [Result1](https://user-images.githubusercontent.com/397672/92924540-5fcb3200-f44e-11ea-93ba-3dc807a65ea6.png), [Result2](https://user-images.githubusercontent.com/397672/92924782-b9cbf780-f44e-11ea-8dc6-93b2be5fb160.png) 

Thanks.

**Comment by dnfield** [comment]
Published on 0.19.0
